cclibs/scripts/scripts.js

```diff
@@ -10,25 +10,29 @@
  * governing permissions and limitations under the License.
  */
 
-import { setLibs } from './utils.js';
-
 // Add project-wide style path here.
-const STYLES = '';
+const STYLES = '/cclibs/styles/styles.css';
 
 // Use '/libs' if your live site maps '/libs' to milo's origin.
-const LIBS = 'https://milo.adobe.com/libs';
+const LIBS = '/libs';
 
 // Add any config options.
 const CONFIG = {
-  // codeRoot: '',
-  // contentRoot: '',
-  // imsClientId: 'college',
-  // geoRouting: 'off',
```

Comment:
nit: you don't need this comment 
`// geoRouting: 'off'`

Reply:
These comments have been removed and are marked as deleted lines. geo routing is turned on as part of fedpub requirement.